Makefile

```diff
@@ -117,14 +118,24 @@ test-appstudio-controller: ## Run test for appstudio-controller only
 ### --- A r g o C D    W e b   U I --- ###
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
 deploy-argocd: ## Install ArgoCD vanilla Web UI
-	$(MAKEFILE_ROOT)/argocd.sh install
+	ARGO_CD_NAMESPACE=$(ARGO_CD_NAMESPACE) ARGO_CD_VERSION=$(ARGO_CD_VERSION) $(MAKEFILE_ROOT)/argocd.sh install
 
 undeploy-argocd: ## Remove ArgoCD vanilla Web UI
-	$(MAKEFILE_ROOT)/argocd.sh remove
+	ARGO_CD_NAMESPACE=$(ARGO_CD_NAMESPACE) ARGO_CD_VERSION=$(ARGO_CD_VERSION) $(MAKEFILE_ROOT)/argocd.sh remove
 
 ### --- F A S T  &  F U R I O U S --- ###
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
 
+install-argocd-openshift: ## Using OpenShift GitOps, install Argo CD to the gitops-service-argocd namespace
+	manifests/openshift-argo-deploy/deploy.sh
+
+install-argocd-k8s: ## (Non-OpenShift): Install Argo CD to the gitops-service-argocd namespace
+	ARGO_CD_VERSION=$(ARGO_CD_VERSION) manifests/k8s-argo-deploy/deploy.sh
+
+uninstall-argocd: ## Uninstall Argo CD from gitops-service-argocd namespace (from either OpenShift or K8s)
+	kubectl delete namespace gitops-service-argocd || true
```

Comment:
Should use the `ARGO_CD_NAMESPACE` env var?

Reply:
I think so as well ^